tests/components/netatmo/test_camera.py

```diff
@@ -253,3 +254,41 @@ async def test_camera_reconnect_webhook(hass, config_entry):
         )
         await hass.async_block_till_done()
         mock_post.assert_called()
+
+
+async def test_webhook_person_event(hass, camera_entry):
+    """Test that person events are handled."""
+    calls = []
+
+    @callback
+    def listener(event):
+        calls.append(event)
+
+    hass.bus.async_listen_once("netatmo_event", listener)
```

Comment:
Use this helper:
https://github.com/home-assistant/core/blob/4fbc3da196650ee587ec023c04fad1cf91d16843/tests/common.py#L1096

Reply:
Ah, great. Thanks for the pointer.